apps/pie-docs/src/tokensMetadata.json

```diff
@@ -989,95 +989,83 @@
     "global": {
       "family-primary": {
         "category": "fontFamily",
-        "displayName": "JET Sans",
-        "description": "Our primary font family"
```

Comment:
sorry if this is dumb, but is there a reason why we're getting rid of the displayName here? the descriptions field  isn't descriptive anymore unless we aren't showing that in designs anymore

Reply:
No wories 😃 
So these `displayNames` were only used on the global tokens. The global tokens designs have now changes so we don't need them any more. It's gone from this ...
<img width="1283" alt="image" src="https://github.com/justeattakeaway/pie/assets/24740015/00d9b56d-263e-4843-854d-c3761e6ab815">
to this
<img width="575" alt="image" src="https://github.com/justeattakeaway/pie/assets/24740015/1d58f25e-52c8-407e-8773-5dc9061afe00">